app/assets/javascript/dropdown_select_all.js

```diff
@@ -0,0 +1,40 @@
+(function($) {
+  "use strict";
+  window.GOVUK = window.GOVUK || {};
```

Comment:
This is maybe a question for frontend devs, but is this module pattern consistent with the rest of the app?

The pattern I've seen in other apps is this one: https://github.com/alphagov/govuk_admin_template/blob/master/JAVASCRIPT.md